lib/svg-sprite/transform/svgo.js

```diff
@@ -70,15 +24,20 @@ var SVGO = require('svgo'),
  */
 module.exports = function (shape, config, spriter, cb) {
     config = _.cloneDeep(config);
-    config.plugins = ('plugins' in config) ? defaultPluginConfig.concat(config.plugins) : defaultPluginConfig;
-    config.plugins.push({ removeXMLProcInst: !!spriter.config.svg.xmlDeclaration });
-    config.plugins.push({ removeDoctype: !!spriter.config.svg.doctypeDeclaration });
+    config.plugins = 'plugins' in config ? svgo.extendDefaultPlugins([...config.plugins]) : [];
```

Comment:
Would be nice to confirm if custom SVGO options still work; we don't have a test for this.